src/plugins/event-plugins/DomEventPlugin.ts

```diff
@@ -84,13 +94,16 @@ export class DomEventPlugin implements Plugin {
         );
     }
 
-    private getEventListener(): EventListener {
+    private getEventListener(domEvent?: TargetDomEvent): EventListener {
```

Comment:
**nit:** We can make this a bit more readable by changing the `domEvent` parameter to `cssLocator`

Reply:
Refactored